Jenkinsfile

```diff
@@ -3,12 +3,16 @@
 import com.cloudogu.ces.dogubuildlib.*
 
 def NAMESPACES = ["testing", "itz-bund", "next", "official"]
+def CREATE_NEW_TAG = "create-new-tag"
+def BUILD_TAG = "build-existing-tag"
+def TAG_STRATEGIES = [CREATE_NEW_TAG, BUILD_TAG]
 
 node('vagrant') {
 
     timestamps {
         def props = [];
-        props.add(string(defaultValue: '', description: 'Dogu Version', name: 'Version', trim: true));
+        props.add(string(defaultValue: '', description: 'Dogu Version', name: 'Version', trim: true))
+        props.add(choice(name: 'Tag_Strategy', choices: TAG_STRATEGIES.join("\n")))
```

Comment:
Why do we need the join? Choices take a list as argument: [Jenkins Documentation](https://www.jenkins.io/doc/book/pipeline/syntax/#available-parameters)

Reply:
What happens on default build, like on push? What is the default strategy? Do we try to push an empty version?